### PR TITLE
Fix Bootsrap install when there is no assets.rb

### PR DESCRIPTION
**Pull request by kivanio**
Fix https://github.com/rails/rails/issues/44597

### CODE REVIEW

**Comment by Edouard-chin** [comment]
I think the fix should be in rails/rails, deleting the `assets.rb` file when using propshaft is initially wrong. Now that Rails supports two asset pipeline out of the box (sprockets/propshaft) we basically need to handle those two cases:

- When an application is using sprockets -> Delete all files specific to propshaft (there is none right now)
- When an application is using propshaft -> Delete all files specific to sprockets (manifest.js I think is the only one)

The code generated in the `assets.rb` initializer is compatible with both propshaft and sprockets and there is no reason to delete it IMO.

**Comment by Edouard-chin** [comment]
Giving this a second thought, it **also** make sense to check if the assets.rb file exists

File: lib/install/bootstrap/install.rb [hunk 1]
```diff
@@ -3,10 +3,12 @@
...
-inject_into_file "config/initializers/assets.rb", after: /.*Rails.application.config.assets.paths.*\n/ do
-  <<~RUBY
-    Rails.application.config.assets.paths << Rails.root.join("node_modules/bootstrap-icons/font")
-  RUBY
+if Rails.root.join('config/initializers/assets.rb').exist?
+  inject_into_file 'config/initializers/assets.rb', after: /.*Rails.application.config.assets.paths.*\n/ do
+    <<~RUBY
+      Rails.application.config.assets.paths << Rails.root.join("node_modules/bootstrap-icons/font")
+    RUBY
+  end
```
Comment (Edouard-chin) [inline lib/install/bootstrap/install.rb:11]: Maybe if it doesn't exist, we can do the same as [below](https://github.com/rails/cssbundling-rails/pull/82/files#diff-6a4335027393c4f283e26e55b5eb99d15469611c3ad6373a32cb4ab28e154ae1L16) and output what to do ?

**Comment by kivanio** [comment]
@Edouard-chin 
I do not know how propshaft works so I assumed assets.rb was not necessary but if it is I think you are right.
It need to be fixed in both places.

File: lib/install/bootstrap/install.rb (hunk 1 above)
Comment (Edouard-chin) [reply lib/install/bootstrap/install.rb:11]: What do you think about

`Add Rails.application.config.assets.paths << Rails.root.join("node_modules/bootstrap-icons/font") in your assets configuration file (by convention in 'config/initializers/assets.rb')`

Comment (kivanio) [reply lib/install/bootstrap/install.rb:11]: Done

Comment (viktorianer) [reply lib/install/bootstrap/install.rb:11]: And why just not create this file `config/initializers/assets.rb`, if it not exists?

**Comment by dhh** [comment]
Fixed via https://github.com/rails/rails/pull/44615

**Pull request closed without merging**